### PR TITLE
Fixed scrollbars and reduced first responder juggling

### LINKED ISSUE #2323: minor side effects of "ScrollView jumping improvements"
["ScrollView jumping improvements" PR](https://github.com/deltachat/deltachat-ios/pull/2315) introduced some side effects that need to be targeted:

- the icon shown on swipe-to-reply is flipped
- ~~tapping the system clock does not scroll _up_ as usual, but scrolls _down_~~ EDIT: might be okay, may be considered even as a fix as we no scroll to the most recent information; the scroll-up in eg. whatsapp is also more random as it scrolls just somewhere to the middle (as more messages are loaded there immediated)
- the scrollbar is gone
- input field is missing after using camera if the _keyboard was not expanded_ before

cc @Amzd

### PR DESCRIPTION
**Pull request by Amzd**
closes #2323

### CODE REVIEW

File: deltachat-ios-uitests/ChatTests.swift
Comment (Amzd) [inline deltachat-ios-uitests/ChatTests.swift:1]: Did not mean to commit this

**Review by r10s — APPROVED** [review]
thanks a lot for taking care! ❤️ this fixes all the issues i discovered after #2315 got merged!

also the code looks excellent and understandable to me, and also makes sense to me.  i suggest to merge this PR as it fixes blocking issues.

for the testing: how can i start this testing in iOS? maybe we can add some hints to the README

File: deltachat-ios-uitests/ChatTests.swift
Comment (r10s) [reply deltachat-ios-uitests/ChatTests.swift:1]: i see :)

is `deltachat-ios.xctestplan` on purpose then?

Comment (Amzd) [reply deltachat-ios-uitests/ChatTests.swift:1]: nope, deleted both from this PR

**Comment by Amzd** [comment]
> for the testing: how can i start this testing in iOS? maybe we can add some hints to the README

That was not intended to be included in this PR, it is not fully done yet. There is some issues I still need to figure out.

**Review by r10s — APPROVED** [review]

**Pull request merged**